src/index.html

```diff
@@ -7,6 +7,73 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
```

Comment:
implements this points
<img width="733" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/102411130/d3042fb4-52dd-4cfb-aed5-4af4ca132fdd">

it should look like this
```
class="calendar  calendar--start-day--sat calendar--month-length--29"